components/automate-ui/src/app/pages/policy/details/policy-details.component.ts

```diff
@@ -74,23 +75,20 @@ export class PolicyDetailsComponent implements OnInit, OnDestroy {
       }));
     this.members$.subscribe();
 
-    this.store.select(routeParams).pipe(
-      pluck('id'),
-      filter(identity),
-      takeUntil(this.isDestroyed))
-      .subscribe((id: string) => {
-        this.store.select(routeURL).pipe(
-          filter(identity),
-          takeUntil(this.isDestroyed))
-          .subscribe((url: string) => {
-            // Only fetch if we are on the policy details route, otherwise
-            // we'll trigger GetPolicy with the wrong input on any route
-            // away to a page that also uses the :id param.
-            if (POLICY_DETAILS_ROUTE.test(url)) {
-              this.store.dispatch(new GetPolicy({ id }));
-            }
-        });
-      });
+    combineLatest([
+      this.store.select(routeParams),
+      this.store.select(routeURL)
+    ]).pipe(
+      takeUntil(this.isDestroyed),
+      map(([params, url]: [Params, string]) => [params.id, url]),
```

Comment:
Until recently I was diligent in advocating to make sure to add explicit types (like you have done here) for these rxjs operators, because there was a typescript bug: type inference was giving them all the same type even if they were not! But it looks like that has been fixed, so now it is redundant. That is, with just this... 
```suggestion
      map(([params, url]) => [params.id, url]),
```
...if you hover over `params` or `url` you will see the correct types inferred.

On the other hand, sometimes it is helpful to provide explicit types if the types are not "obvious", so it is a personal preference.

But on the third hand, this code block uses that pattern four times, which makes the block more busy; it would be more readable if you eliminate all four occurrences.

And finally, if doing so, this line really needs to become:
```suggestion
      map(([params, url]) => [params.id as string, url]),
```
Because `params.id` is of type `any`.

Reply:
I am not seeing that the types are inferred in VS code.

Reply:
I am seeing it now. 🤷‍♂